docs/docs-beta/docs/guides/build/project-structure/existing-project.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Making an existing project components-compatible"
+sidebar_position: 100
+---
+
+:::note
+This guide is only relevant if you are starting from an *existing* dagster project. This setup is unnecessary if you used `dg code-location generate` to create your project.
+:::
+
+## Create a `components` directory
+
+First, you'll want to create a directory to contain any new components you add to your project. By convention, this directory is named `components`, and exists at the top level of your code location.
```

Comment:
nit: top level of your code location's python module, right? since it's e.g. `jaffle_shop/jaffle_shop/components` where `jaffle_shop/pyproject.toml` exists